Packs/GoogleChronicleBackstory/Integrations/GoogleChronicleBackstory/GoogleChronicleBackstory.yml

```diff
@@ -554,7 +554,1212 @@ script:
     - contextPath: GoogleChronicleBackstory.Alert.AlertCounts
       description: The total number of alerts.
       type: Number
-  dockerimage: demisto/googleapi-python3:1.0.0.7535
+  - arguments:
+    - auto: PREDEFINED
+      default: false
+      description: Specify the identifier type of the asset you are investigating.
+        The possible values are Host Name, IP Address, MAC Address or Product ID.
+      isArray: false
+      name: asset_identifier_type
+      predefined:
+      - Host Name
+      - IP Address
+      - MAC Address
+      - Product ID
+      required: true
+      secret: false
+    - default: false
+      description: Value of the asset identifier.
+      isArray: false
+      name: asset_identifier
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Get events that are discovered during the interval specified. If
+        configured, overrides the start_time and end_time arguments.
+      isArray: false
+      name: preset_time_range
+      predefined:
+      - Last 1 day
+      - Last 7 days
+      - Last 15 days
+      - Last 30 days
+      required: false
+      secret: false
+    - default: false
+      description: The value of the start time for your request. The format of Date
+        should comply with RFC 3339 (e.g. 2002-10-02T15:00:00Z). If not supplied,
+        the product considers UTC time corresponding to 2 hours earlier than current
+        time.
+      isArray: false
+      name: start_time
+      required: false
+      secret: false
+    - default: false
+      description: The value of the end time for your request. The format of Date
+        should comply with RFC 3339 (e.g. 2002-10-02T15:00:00Z). If not supplied,
+        the product considers current UTC time.
+      isArray: false
+      name: end_time
+      required: false
+      secret: false
+    - default: false
+      defaultValue: '10000'
+      description: Specify the maximum number of events to fetch. You can specify
+        between 1 and 10000. The default is 10000.
+      isArray: false
+      name: page_size
+      required: false
+      secret: false
+    - default: false
+      description: Specify the reference time for the asset you are investigating,
+        in RFC 3339 format (e.g. 2002-10-02T15:00:00Z). If not supplied, the product
+        considers start time as reference time.
+      isArray: false
+      name: reference_time
+      required: false
+      secret: false
+    deprecated: false
+    description: List all of the events discovered within your enterprise on a particular
+      asset within the specified time range.
+    execution: false
+    name: gcb-list-events
+    outputs:
```

Comment:
does the users needs this all 1000 line output per event?

Reply:
we have 68 types of events and each event have different key in JSON response which make huge contextData information in yml.